webviz_subsurface/plugins/_inplace_volumes_onebyone.py

```diff
@@ -24,53 +24,72 @@
 
 class InplaceVolumesOneByOne(WebvizPluginABC):
     # pylint: disable=too-many-instance-attributes
-    """### InplaceVolumesOneByOne
+    """Visualizes inplace volumetrics related to a FMU ensemble with a design matrix.
 
-Visualizes inplace volumetrics related to a FMU ensemble with design matrix.
+Input can be given either as an aggregated `csv` file for volumes and sensitivity information,
+or as ensemble name(s) defined in `shared_settings` and volumetric `csv` files
+stored per realization.
 
-Input can be given either as aggregated csv files for volumes and sensitivity information,
-or as an ensemble name defined in *shared_settings* and volumetric csv files
-stored per realizations.
+---
 
-#### Volumetric input
-The volumetric csv files must follow FMU standards.
-[Example csv file](
-https://github.com/equinor/webviz-subsurface-testdata/blob/master/aggregated_data/volumes.csv)
+* **`csvfile_vol`:** Aggregated csvfile for volumes with `REAL`, `ENSEMBLE` and `SOURCE` columns.
+* **`csvfile_parameters`:** Aggregated csvfile of parameters for sensitivity information with
+`REAL`, `ENSEMBLE`, `SENSNAME` and `SENSCASE` columns.
+* **`ensembles`:** Which ensembles in `shared_settings` to visualize (not to be used with
+`csvfile_vol` and `csvfile_parameters`).
+* **`volfiles`:**  Key/value pair of csv files when using `ensembles`.
+E.g. `{geogrid: geogrid--oil.csv}`.
```

Comment:
Do we need `\` here?

Reply:
Not really, you need a whiteline for it to be read as a new line. Think I've rather overused than underused the \. On the other hand, it feels clearer that it is a part of the bullet point if adding a \, so think I will do that 👍